cfep-03.md

```diff
@@ -0,0 +1,51 @@
+
+<table>
+<tr><td> Title </td><td> Providing manually uploaded builds on the conda-forge anaconda channel</td>
+<tr><td> Status </td><td> Proposed </td></tr>
+<tr><td> Author(s) </td><td> Hadrien Mary &lt;hadrien.mary@gmail.com&gt;</td></tr>
+<tr><td> Created </td><td> Sept 27, 2016</td></tr>
+<tr><td> Updated </td><td> Sept 27, 2016</td></tr>
+<tr><td> Discussion </td><td> NA </td></tr>
+<tr><td> Implementation </td><td> NA </td></tr>
+</table>
+
+## Abstract
+
+Some packages are very hard to build on the currently used CI (circleci, Travis and AppVeyor). It's most of the time because of time limitations in the CI website and also because of the complexity of the build.
+
+An example is the Qt5 build that seems to be very hard to build (it tooks almost a year for Continuum folks to compile it for all platforms). See the Qt5 recipe build discussion ticket here https://github.com/conda-forge/qt-feedstock/issues/5.
+
+That being said it's excepted that a small amount of packages will be very hard to automatically build for various reasons. 
+
+Obviously we all want to keep this amount of packages small and have a maximun of automatically build packages on conda-forge.
+
+But a system/protocol to manually upload builds on conda-froge anaconda channel is necessary to circumvent some situations.
+
+## Specification
+
+A protocol to manually upload builds could look something like that :
+
+- Manually build the different packages for all targeted platforms.
+- Test for each platform on different version (multiple Linux distributions, Windows 7/10, etc).
+- Someone from the @conda-forge/core (having the permissions to upload on the conda channel) use conda command line to upload the package.
+- Build is available in the conda-forge anaconda channel.
+
+**Open Ideas/Questions**
+
+- Could we use CI to test manually uploaded packages ? How ?
+- How the package is transfered from the user to the @conda-forge/core person before the upload ?
+
+## Motivation
+
+Main motivation is to be able to provide the widest range of possible packages in conda-forge. This proposal will be really usefull to upload very hard to build packages.
+
+## Alternatives
+
+A CI infrastructure that build them all !
+
+## FAQ
+
+
+## Copyright
+
+This document has been placed in the public domain.
```

Comment:
Please change to `This document is CC0 1.0 Universal.` Not everywhere has Public Domain, but this is the next best thing. IANAL